compiler-rt/lib/radsan/tests/radsan_test_interceptors.cpp

```diff
@@ -33,6 +33,21 @@ using namespace std::chrono_literals;
 
 namespace {
 void *fake_thread_entry_point(void *) { return nullptr; }
+
+/*
+  The creat function doesn't seem to work on an ubuntu Docker image when the
+  path is in a shared volume of the host. For now, to keep testing convenient
+  with a local Docker container, we just put it somewhere that's not in the
+  shared volume (/tmp). This is volatile and will be cleaned up as soon as the
+  container is stopped.
+*/
+constexpr auto temporary_file_path() -> const char * {
```

Comment:
Any reason for the trailing return type? Does that match the rest of llvm's style? (Honest question, I don't know)

Reply:
Ah yeah - good question. I did initially play with `constexpr const char * temporary_file_path() {`, but looked at it and thought the trailing return type was easier to understand! We've got a bit of a mixture of styles in here anyway, which isn't great (e.g. snake case `fake_thread_entry_point()` and `temporary_file_path()` vs. camel case `expectRealtimeDeath()` and `expectNonrealtimeSurvival()`). There doesn't seem to be really tight style control in the compiler-rt project (the Google-developed sanitizers have Google style, for example, which is a little different to LLVM style). Not sure if we need to worry about it yet!